themes-default/slim/static/js/store.js

```diff
@@ -448,19 +450,20 @@ const store = new Puex({
             return api.patch('config/' + section, config).then(setTimeout(() => dispatch('getConfig'), 500));
         },
         getShow(context, { indexer, id }) {
+            const { commit } = context;
             return api.get('/series/' + indexer + id).then(res => {
-                store.commit(ADD_SHOW, res.data);
+                commit(ADD_SHOW, res.data);
```

Comment:
Sorry if I caused it, but on schedule page this error pops up:
```
Uncaught (in promise) TypeError: Cannot read property 'mutations' of undefined
    at commit (puex.js:136)
    at shows.forEach.show (store.js:466)
    at Array.forEach (<anonymous>)
    at api.get.then.res (store.js:465)
```

Reply:
Try the latest commit I pushed. I think I fixed this.

Reply:
Yes, looks good now, I'll sync these to the other themes.

By the way, schedule is now a bit outdated. I'll fix that.